### PR TITLE
chore: remove todos

### PR DESCRIPTION
**Pull request by kabaros**

### CODE REVIEW

File: cli/src/lib/paths.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ module.exports = (cwd = process.cwd(), { typeScript } = {}) => {
             './.d2/shell/src/plugin.index.jsx'
         ),
         shellPluginEntrypoint: path.join(base, './.d2/shell/src/Plugin.jsx'),
-        // TODO: look at what to do with service-worker files whether to move to TS or not
+
```
Comment (kabaros) [inline cli/src/lib/paths.js:112]: removed this todo - it's a general question about whether we should move this whole repo to TypeScript, not the place for it in this PR

File: cli/src/lib/paths.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ module.exports = (cwd = process.cwd(), { typeScript } = {}) => {
         package: path.join(base, './package.json'),
         yarnLock: findYarnLock(base, rootDirectory),
         dotenv: path.join(base, './.env'),
-        config: path.join(base, './d2.config.js'), // TODO: double check this works
+        config: path.join(base, './d2.config.js'),
```
Comment (kabaros) [inline cli/src/lib/paths.js:88]: checked it works

File: cli/src/commands/build.js [hunk 3]
```diff
@@ -61,7 +61,6 @@ const handler = async ({
     pack: packAppOutput,
     allowJsxInJs,
 }) => {
-    // todo: we need to infer TypeScript in build command here similar to start
     const paths = makePaths(cwd)
```
Comment (kabaros) [inline cli/src/commands/build.js:64]: checked the build command works

**Review by KaiVandivier — APPROVED** [review]
Nice thanks!

**Pull request merged**